modules/aws/ebs/main.tf

```diff
@@ -0,0 +1,75 @@
+/**
+ * AWS EBS
+ * =======
+ * This module creates additional ebs volumes and attaches them to the instances.
+ *
+ * EXAMPLE
+ * -------
+ *
+ *```hcl
+ * module "instance-volume" {
+ *   source = "dcos-terraform/ebs/aws"
+ *
+ *   instances          = ["id-ablsldhfa123", "id-aidrkdkek131"]
```

Comment:
in generally do not like the idea of inter module resource changes. Volume management should be part of the instance module not an external one.

All modules are currently design to keep this approach and don't do intermodule changes. We should prefer buiding a similar solution in the instance module. I would only agree on such a solution if there is no useful way doing it with the instance module

Reply:
if this should be a feature of the universal installer yes.

Reply:
OK, sgtm!